### DISCUSSION TITLE
mounted storage only has root access

### DISCUSSION THREAD

**Discussion by Magg06** [trapexit/mergerfs#1372] [Q&A]
Hi,
I'm new to it not sure what I'm doing wrong. 
At the moment I have 2 drives, one for storage and one parity, Will add one more to storage but need to copy files from it first.
I'm on Ubuntu and have mergerfs v2.40.2. I have the storage in /mnt/storage but it has only root access so I can't create any folders there if I don't do it from terminal with sudo. Do I need to change something in Ubuntu or I configured mergerfs incorrectly.

**Comment by trapexit** [comment]
mergerfs is just a proxy for the already existing permissions. You simply need to change the mode and ownership appropriately.

**Reply by Magg06** [reply]
thanks, where can I read about it. wherever I look whatever solutions are given cant get it to work and change from root

**Reply by trapexit** [reply]
"new" meaning you don't know how Linux/Unix works? You just use "chown" and "chmod".

**Reply by trapexit** [reply]
https://www.google.com/search?q=how+Linux+filesystem+permissions+work

**Reply by Magg06** [reply]
I've tried chmod as that was what everyone was saying and it didn't do anything. Tried chown and managed to get it to work. Thank you. Yeah I know how it works a bit but didn't touch it in 10 years or so. Sorry for stupid questions, and thank you for help

**Reply by trapexit** [reply]
Without you saying what the state of the file permissions are and what you are doing it's impossible to tell you what is wrong.

**Reply by Magg06** [reply]
I manage to get it working, had to change ownership from root to my user. thanks again
